### PR TITLE
Track main branch for official Chromium build instructions

### PR DESCRIPTION
**Pull request by tangalbert919**
**tl;dr:** Updating the link to Chromium's official build instructions per major update is a nuisance, so just track their main branch instead. Google requires Windows 10 now, so update our instructions to reflect that.

**Long version:** Whenever there's a new major update for Chromium, we have to update the link to the official build instructions in the README. Usually, the build instructions don't change per update, but because of this, having to update the link is a nuisance. Plus, we might forget to update the link (the last time it was updated was when version 89 came out in March).

So instead of doing that, we'll just have the link point to the official instructions on the main branch. It's either that or update the link every four weeks.

In addition to that, Google now requires using Windows 10 or newer to build, so this PR will also update the instructions in the README to reflect such a change.

### CODE REVIEW

**Review by networkException — APPROVED** [review]
Oh I didn't even notice that the urls were pinned to a tag

**Review by Eloston — APPROVED** [review]
So I made this a tag initially because it allows us to stay consistent in case the instructions update before we have a chance to update our fork. But I see now that there's no use-case for this (we almost never want to build old versions of the browser, and we always try to keep up-to-date), so I'm okay to point this to the main branch.

I think there can still be an issue where the instructions from main don't apply to the latest stable (e.g. tooling changes), but I don't think that's a very common occurrence.
